docs/release-notes/v1.1.rst

```diff
@@ -1,6 +1,178 @@
+*gammapy.visualization*
+
+- A function to plot `Map` as RGB is now proposed: `~gammapy.visualization.plot_map_rgb()`.
+- A function to plot the spectral distributions of predicted counts of models defined
+  on a `Dataset` is now available : `~gammapy.visualization.plot_npred_signal()`
+
+*documentation*
+
+- A new tutorial demonstrate how to use Gammapy with HAWC data.
+- The pulsar analysis tutorial now shows how to perform 3D analyses in phase.
+- A new tutorial demonstrating event sampling with energy dependent temporal models has
+  been added.
+
+*infrastructure*
+
+- Deprecation warnings can now be raised by Gammapy deprecated code. The warnings will appear
+  in the documentation and docstrings as well. Deprecated features will usually be removed
+  in the following feature release.
+- Minimal python version is now 3.9. Python 3.10 is supported as well.
+
+API changes
+~~~~~~~~~~~
+
+*gammapy.data*
+
+- the `GTI` now consists in a simple `Table` with `START` and `STOP` as `~astropy.time.Time`
+  object. `GTI.table` no longer contains a GADF formatted table with columns representing
+  start and stop time as METs (Mission Elpased Time). All methods should behave equivalently
+  with the same interface.
+- `~gammapy.irf.load_cta_irfs` is now deprecated. Use instead the more general
+  `~gammapy.irf.load_irf_dict_from_file`
+
+*gammapy.utils*
+
+- The `~gammapy.utils.table.table_from_row_data()` is now deprecated. It can be simply replaced
+  by the regular constructor : `Table(rows)`.
+
+Bug fixes and improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- The energy dependent RADMAX cut is now supported by the `~gammapy.makers.PhaseBackgroundMaker`.
+- The `~gammapy.estimators.TSMapEstimator` now accepts `MapDatasetOnOff` as well
+  as regular `MapDataset`.
+
+
+Contributors
+~~~~~~~~~~~~
+
+- Arnau Aguasca
+- Axel Donath
+- Luca Giunti
+- Lucas Gréaux
+- Bruno Khélifi
+- Maximilian Linhoff
+- Cosimo Nigro
+- Laura Olivera-Nieto
+- Fabio Pintore
+- Maxime Regeard
+- Quentin Rémy
+- Atreyee Sinha
+- Katrin Streil
+- Régis Terrier
+
+Pull Requests
+~~~~~~~~~~~~~
+
+This list is incomplete. Small improvements and bug fixes are not listed here.
+
+- [#4510] Introduce WcsNDMap.cutout_and_mask_region (Axel Donath)
+- [#4508] Implement WcsNDMap.to_region_nd_map_histogram (Axel Donath)
+- [#4506] Rename append method of MapAxis and LabelMapAxis to concatenate (Maxime Regeard)
+- [#4495] Introduce move_axis method on Map (Régis Terrier)
+- [#4466] Add tutorial for the use of HAWC data (Laura Olivera-Nieto)
+- [#4459] Evaluation of energy dep temporal model (Atreyee Sinha)
+- [#4444] Integral sensitivity in FluxPointsEstimator (Atreyee Sinha)
+- [#4442] Add stack option to npred_signal() (Maxime Regeard)
+- [#4430] Add squash method to LabelMapAxis (Maxime Regeard)
+- [#4428] Add .to_string() to axis y/xlabel (Arnau Aguasca)
+- [#4427] Add example for on-off simulation (Atreyee Sinha)
+- [#4417] adding from_stack and append to LabelMapAxis (Maxime Regeard)
```

Comment:
4418 should feature in this list, no?

Reply:
Good point. It will. It was not yet merged when I started the PR.